tools/barista/src/utils/isPublicBuild.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function isPublicBuild(): boolean {
+  return process.env.PUBLIC_BUILD === 'true';
```

Comment:
@tomheller does it make sense to read it once and store it in a var so we do not have to access the `process.env` every time

Reply:
I don't think this is necessary. The access to the process environments is rather quick, and as long as we have the access only in one place (as we do now) I don't think we need to bother with caching the value in memory.